js/adapt-contrib-slider.js

```diff
@@ -371,10 +397,13 @@ define(function(require) {
 
         // according to given item index this should make the item as selected
         selectItem: function(itemIndex) {
+            this.$el.a11y_selected(false);
             _.each(this.model.get('_items'), function(item, index) {
                 item.selected = (index == itemIndex);
                 if(item.selected) {
                     this.model.set('_selectedItem', item);
+                    this.$('.confidenceSlider-scale-number[data-id="'+(itemIndex+1)+'"]').a11y_selected(true);
```

Comment:
Thanks Ollie,

I think Slider and Confidence Slider are not tightly coupled as we have done for HotGraphic, who transform into Narrative on small screen devices. Also keeping the code clean and maintainable, we should make changes to relative components so that we could assure by not making any hardcore dependencies.

Sorry, but let me correct you here about ConfidenceSlider do override [selectItem](https://github.com/adaptlearning/adapt-contrib-confidenceSlider/blob/develop/js/adapt-contrib-confidenceSlider.js#L231-L240).